app.js

```diff
@@ -109,9 +119,9 @@ define(function(require) {
 				addBackButton: true,
 				noFocus: true,
 				onNewAccountClick: function(parentAccountId, breadcrumbs) {
-					self.renderNewAccountWizard({
-						parent: parent,
-						accountId: parentAccountId || self.accountId
+					monster.pub('accounts.newAccountWizard.render', {
+						container: parent,
+						parentAccountId: parentAccountId || self.accountId
```

Comment:
I am wondering how submodule methods should be called by the parent app...

I remember telling you to use our pub/sub system to invoke them but now I am wondering if we shouldn't follow this scheme:

1. only publish methods when they come from different apps
2. invoke the method directly when calling a submodule method from it's main app (with the exception of common controls: since they're not really related to each other, we should always default to use the first option)

The rationale behind this is to avoid overloading the message bus with unnecessary event subscriptions since submodule methods are merged into the app object and accessible like any regular app methods.

I'll try to improve the doc regarding pub/sub so we have a clearly defined strategy when it comes to this.

Reply:
I agree with the scheme, because the method is not intended to be public, and it will be available inside the main app so it can be invoked directly. In this case, however, I'll keep the event, because I just noticed that there is a shortcut to launch the account creation page, and it must be related to an event name to work.

Reply:
Ah yes, seems like that event is tied to a keyboard shortcut.